tasks/main.yml

```diff
@@ -7,6 +7,11 @@
 - name: Include OS variables
   include_vars: "{{ ansible_os_family }}.yml"
 
+- name: Set nomad_servers to empty list rather than the default macro if we are using consul
```

Comment:
Is this needed? Seems like extra code that doesn't really make a difference.

Just ignoring the macro if `nomad_use_consul = true` seems a better option to me. (As this PR does in the template code)